src/components/search/curated/[slug]/index.tsx

```diff
@@ -156,94 +261,125 @@ const CuratedTopic: React.FC<CuratedTopicProps> = ({topic, topicData}) => {
-                  </Grid>
+                        })}
+                      </div>
+                    </div>
+                  )
+                case section.resources.length === 4 && !section.image:
+                  let primarySections = section.resources.slice(0, 2)
+                  let secondarySections = section.resources.slice(2, 4)
+
+                  return (
+                    <div>
+                      <div className="flex md:flex-row flex-col md:items-start items-center justify-start w-full mb-5 pb-8 md:space-x-10">
+                        <div className="grid lg:grid-cols-4 gap-4">
+                          <div className="col-span-2">
+                            <h2 className="w-full lg:text-2xl sm:text-xl text-lg dark:text-white font-semibold leading-tight pb-4">
+                              {section.title}
+                            </h2>
+                            {section.description && (
+                              <ReactMarkdown className="prose sm:prose prose-sm dark:prose-dark dark:text-gray-300 text-gray-700 dark:prose-sm">
+                                {section.description}
+                              </ReactMarkdown>
+                            )}
+                          </div>
+                          {secondarySections.map((resource: any) => {
+                            return (
+                              <div className="hidden lg:block self-center">
+                                <VerticalResourceCard
+                                  key={resource.title}
+                                  resource={resource}
+                                  location={location}
+                                  completedCoursesIds={completedCoursesIds}
+                                />
+                              </div>
+                            )
+                          })}
+                          {primarySections.map((resource: any) => {
+                            return (
+                              <HorizontalResourceCard
+                                className="col-span-2  hidden lg:block"
+                                key={resource.title}
+                                resource={resource}
+                                location={location}
+                                completedCoursesIds={completedCoursesIds}
+                              />
+                            )
+                          })}
+                          {section.resources.map((resource: any) => {
+                            return (
+                              <VerticalResourceCard
+                                className="lg:hidden"
+                                key={resource.title}
+                                resource={resource}
+                                location={location}
+                                completedCoursesIds={completedCoursesIds}
+                              />
+                            )
+                          })}
```

Comment:
When the screen fits the content, we split the resources and display 2 courses as horizontal cards and 2 as vertical. When it is smaller we go to the default which is vertical for all 4